3.9-minimal/test/run

```diff
@@ -220,7 +220,8 @@ test_from_dockerfile(){
   check_result $?
 
   # mod_wsgi in micropipenv-requirements-test-app
-  sed "s@#IMAGE_NAME#@${IMAGE_NAME}@" $test_dir/from-dockerfile/mod_wsgi.Dockerfile.tpl > $test_dir/from-dockerfile/Dockerfile
+  # TODO: Find a better way how to specify full/minimal images that should work together
+  sed "s@#IMAGE_NAME#@${IMAGE_NAME}@;s@#FULL_IMAGE_NAME#@ubi8/python-39:1@" $test_dir/from-dockerfile/mod_wsgi.Dockerfile.tpl > $test_dir/from-dockerfile/Dockerfile
```

Comment:
This seems a bit hardcoded. How about creating a `FULL_IMAGE_NAME` variable similar to `IMAGE_NAME`?

Reply:
It would be nice to have something like that in the future but I don't see an easy way how to do it now. How to find a proper full-size container if you only know the name of the minimal one? Some regex or replace might work in the future and it will be a part of the distgen implementation. It can then also be implemented as a distgen variable that every minimal image will have the name of the full-size image defined manually.

Reply:
I like the distgen variable approach. I see there's a TODO note for the future, so I'm fine with leaving it as it is for now.